src/components/Form/Checkbox.tsx

```diff
@@ -127,6 +127,15 @@ export class Checkbox extends Component<CheckboxProps> {
       );
     }
 
+    if (
+      ('name' in this.props && name === '') ||
```

Comment:
need to test for undef and null as well?

Reply:
Good point. Should now cover those as well.